src/printer.js

```diff
@@ -3375,16 +3387,18 @@ function printJSXChildren(path, options, print, jsxWhitespace) {
         // whitespace(s)-only without newlines,
         // eg; one or more spaces separating two elements
         for (let i = 0; i < value.length; ++i) {
-          children.push(jsxWhitespace);
+          children.push(innerJsxWhitespace);
```

Comment:
Oops, I attempted to fix the merge conflict but it didn't work, it's now breaking many tests